### PR TITLE
Correct errors from CATS report

### PR DESCRIPTION
**Pull request by smitpatel**
- Missing alerts

Also bunch of warnings from authoringpack in VS.

### CODE REVIEW

File: entity-framework/core/get-started/index.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ You can follow the tutorial by using Visual Studio on Windows, or by using the .
 
 Install the following software:
 
-# [.NET Core CLI](#tab/netcore-cli)
+## [.NET Core CLI](#tab/netcore-cli)
```
Comment (roji) [inline entity-framework/core/get-started/index.md]: Wasn't aware, thanks!

File: entity-framework/core/modeling/relational/indexes.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Indexes - EF Core
+title: Indexes (Relational Database) - EF Core
 author: rowanmiller
 ms.date: 10/27/2016
 ms.assetid: 4581e7ba-5e7f-452c-9937-0aaf790ba10a
 uid: core/modeling/relational/indexes
 ---
-# Indexes
+# Indexes (Relational Database)
```
Comment (roji) [inline entity-framework/core/modeling/relational/indexes.md]: Am going to unify the two index pages but OK for now.

File: entity-framework/core/get-started/index.md (hunk 1 above)
Comment (smitpatel) [reply entity-framework/core/get-started/index.md]: This also shows warning. H2 with same title multiple times. But multiple H1 is more severe error. I will also test out not put it as heading at some point.

Comment (roji) [reply entity-framework/core/get-started/index.md]: The issue here is probably that the tab syntax is a docfx extension, this is just how it works... It's weird for multiple H2 with the same title to flag an issue even without tabs, maybe we can suppress certain issues.

**Pull request merged**